src/WPF/ArcGISRuntime.WPF.Viewer/MainWindow.xaml.cs

```diff
@@ -185,6 +189,13 @@ private async Task SelectSample(SampleInfo selectedSample)
 
             CategoriesRegion.Visibility = Visibility.Collapsed;
             SampleContainer.Visibility = Visibility.Visible;
+            SetScreenshotButttonVisibility(ScreenshotManager.ScreenshotSettings.ScreenshotEnabled);
+            SetContainerDimensions(ScreenshotManager.ScreenshotSettings.ScreenshotEnabled);
+        }
+
+        private void SetScreenshotButttonVisibility(bool isEnabled)
```

Comment:
Parameter can be removed, just reference the screenshot manager from within this method.